### ISSUE TITLE
Does this apply to K1C

### ISSUE THREAD

**Issue by tent228** [ballaswag/creality_k1_klipper_mod#3]
Does this apply to K1C？

**Comment by ballaswag** [comment]
Yes it has native support for the K1C. Do try with care since I have only tested on the K1.

**Comment by tent228** [comment]
Thank you very much. I have installed and used this mod in K1C, but there are a few issues.
1. I used to connect to the printer by filling in the fluidd's IP and port in Orcaslicer, but now I have to change it to the it to the port of Moonraker. Similarly, I can only connect to Moonraker using KNOMI, and previously I used to directly connect to Fluidd.
2. I am using nginx proxy manager to set up a reverse proxy connection to Fluidd. I am prompted to enter my username and password, but when I enter it, I receive a 501 error message. Can you please solve this problem?

**Comment by Rawend1** [comment]
Hello @tent228 @ballaswag  folks. 

I have strange issue after installing on K1C: "mcu 'eddy': Unable to connect". My steps was:
Root creality
Install fluidd and other components. 
Recover to original firmware then update to last official version
Rooted and Installed this firmware . Reboot printer.
It shows Fluidd on port 4001, but shows message "mcu 'eddy': Unable to connect" and printer doesn't initializing. Screen shows message "waiting for printer to initialize"

I added into printer.cfg strings
[mcu rpi]
serial: /tmp/klipper_host_mcu
[duplicate_pin_override]
pins: PC0, PC5

like printer had before flushing new firmware, but it did not help.

Do you have any suggestions how to fix printer? I didn't tried yet to get back to official firmware.

UPD 2025-10-21
Thanks for roll-back function, I get to previous official firmware. It works.
